pages/project/read/views/index.jsx

```diff
@@ -68,13 +68,13 @@ export default function ProjectLandingPage() {
     downloads: <Snippet>{`${snippetPath}.downloads`}</Snippet>
   };
 
-  const { additionalAvailability, showManageSection } = useSelector(state => state.static);
+  const { additionalAvailability, showManageSection, canUpdate } = useSelector(state => state.static);
 
   if (additionalAvailability || (!hasHistory() && !hasPreviousVersions())) {
     delete sections.history;
   }
 
-  if (project.status === 'inactive') {
+  if (project.status === 'inactive' && canUpdate) {
```

Comment:
```suggestion
  if (project.status === 'inactive' || !canUpdate || !asruLicensing) {
```

Reply:
This will need the `asruLisensing` bit putting back to wherever that was before.